.github/workflows/cd.yml

```diff
@@ -26,140 +26,93 @@ jobs:
+          - name: linux-x86-glibc
+            os: ubuntu-latest
+            target: i686-unknown-linux-gnu
+            publish_npm: false
+            publish_pypi: false
+          - name: linux-x86-musl
+            os: ubuntu-latest
+            target: i686-unknown-linux-musl
+            publish_npm: false
+            publish_pypi: true
+            install_musl: true
+          - name: linux-arm64-glibc
+            os: ubuntu-24.04-arm
+            target: aarch64-unknown-linux-gnu
+            publish_npm: true
+            publish_pypi: true
+          - name: linux-arm64-musl
+            os: ubuntu-24.04-arm
+            target: aarch64-unknown-linux-musl
+            publish_npm: false
+            publish_pypi: true
+            install_musl: true
+          - name: win32-x64-mingw
+            os: windows-latest
+            target: x86_64-pc-windows-gnu
+            publish_npm: false
+            publish_pypi: false
+          - name: win32-x64-msvc
+            os: windows-latest
+            target: x86_64-pc-windows-msvc
+            publish_npm: true
+            publish_pypi: true
+          - name: win32-x86-msvc
+            os: windows-latest
+            target: i686-pc-windows-msvc
+            publish_npm: false
+            publish_pypi: true
+          - name: win32-arm64-msvc
+            os: windows-11-arm
+            target: aarch64-pc-windows-msvc
+            publish_npm: true
+            publish_pypi: false
+          - name: darwin-x64
+            os: macos-13
+            target: x86_64-apple-darwin
+            publish_npm: true
+            publish_pypi: true
+          - name: darwin-arm64
+            os: macos-latest
+            target: aarch64-apple-darwin
+            publish_npm: true
+            publish_pypi: true
     steps:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Set the release version
         shell: bash
         run: echo "RELEASE_VERSION=${GITHUB_REF:11}" >> $GITHUB_ENV
-      - name: Install dependencies
+      - name: Install Musl tools
+        if: matrix.install_musl
         shell: bash
         run: |
-          if [[ "${{ matrix.build.NAME }}" = *"-musl" ]]; then
-            sudo apt-get update
-            sudo apt-get install -y --no-install-recommends \
-              --allow-unauthenticated musl-tools
-          fi
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends --allow-unauthenticated \
+            musl-tools
       - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{ matrix.build.TOOLCHAIN }}
-          target: ${{ matrix.build.TARGET }}
-          override: true
-      - name: Build (linux/macos)
-        if: matrix.build.OS != 'windows-2022'
-        uses: actions-rs/cargo@v1
-        with:
-          use-cross: true
-          command: build
-          args: --release --locked --target ${{ matrix.build.TARGET }}
```

Comment:
I think cross is no longer necessary because the arm stuff is being built on the right runner now.

I'd suggest doing a quick check to validate this by turning off all the release parts of this change in another PR and just checking that all the build succeeds.

Reply:
I'm not so sure about not needing `cross`. This was extremely fragile to put together and testing it takes a lot of time.

Reply:
Checking this in https://github.com/orhun/git-cliff/pull/1198